### PR TITLE
Quiet down integration test logging

### PR DESCRIPTION
**Pull request by dturanski**
Add logback.xml to set TestContainers logging level.

Add spring-boot-geode dependency management for geode apps, required for `spring-geode-starter-logging` which fixes the logging config.

### CODE REVIEW

File: applications/sink/geode-sink/src/test/resources/logback-include.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+		</encoder>
+	</appender>
+
+	<appender name="delegate" class="org.springframework.geode.logging.slf4j.logback.DelegatingAppender"/>
+
+	<logger name="com.gemstone.gemfire" level="${spring.boot.data.gemfire.log.level:-WARN}"/>
+	<logger name="org.apache.geode" level="${spring.boot.data.gemfire.log.level:-WARN}"/>
+	<logger name="org.jgroups" level="${spring.boot.data.gemfire.jgroups.log.level:-ERROR}"/>
+
+</included>
```
Comment (onobc) [inline applications/sink/geode-sink/src/test/resources/logback-include.xml:16]: All of the logback-include.xml have no newline at end. Github lighting that up.

File: applications/sink/geode-sink/src/test/resources/logback-include.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+	<appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<pattern>%d %5p %40.40c:%4L - %m%n</pattern>
+		</encoder>
+	</appender>
+
+	<appender name="delegate" class="org.springframework.geode.logging.slf4j.logback.DelegatingAppender"/>
+
+	<logger name="com.gemstone.gemfire" level="${spring.boot.data.gemfire.log.level:-WARN}"/>
```
Comment (onobc) [inline applications/sink/geode-sink/src/test/resources/logback-include.xml:12]: Is the property key something specific to the Geode starter or did we create it? We could instead just set a default level and then recommend the standard [Spring Boot method](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#features.logging.log-levels) of adjusting the log level settings.

**Review by onobc — COMMENTED** [review]
@dturanski LGTM - thanks for doing this. 

I left a couple comments for you.

File: applications/sink/geode-sink/src/test/resources/logback-include.xml (hunk 2 above)
Comment (dturanski) [reply applications/sink/geode-sink/src/test/resources/logback-include.xml:12]: This is probably cut and paste from an example somewhere.  The geode apps introduced a dependency on `spring-boot-data-gemfire` which came subsequent to these apps. Geode uses proprietary logging config, so it's got its own property now.  I noticed the pattern is different than the others. I don't have a strong opinion on this.

File: applications/sink/mqtt-sink/src/test/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<configuration>
+
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
```
[the remark this answers is not in the document]
Comment (dturanski) [reply applications/sink/mqtt-sink/src/test/resources/logback.xml:5]: They are the same now

**Comment by dturanski** [comment]
OK. Reverted the timesource metadata changes. Should be good to go.

**Review by onobc — APPROVED** [review]

**Comment by onobc** [comment]
Closed via https://github.com/spring-cloud/stream-applications/commit/b8cb30fa9ea8d8b9a5ce2ec9e848fb41b243291c

**Pull request closed without merging**